### DISCUSSION TITLE
Hello, I have used Alipay to renew but my professional version account information is still not updated.

### DISCUSSION THREAD

**Discussion by farhigh233** [getlantern/lantern-forum-cn#662] [Q&A] [answered]
Hello, I have used Alipay to renew but my professional version account information is still not updated.

My account number is [231879414@qq.com]

Please check it for me, thank you @Mike777Khoo

**Comment by MaxMKMax** [comment, marked as answer]
@farhigh233 Hi user, in order to protect your privacy information. Can you submit the issues via this link? Provide your email address and payment receipt via the link. Thank you for supporting Lantern. Appreciate it so much.

https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ

**Reply by farhigh233** [reply]
I have sent the email address and the payment receipt to this link
